x/ibc-hooks/contracts/swaprouter/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+# This feature has been stabilized in 1.64. If using an earlier version (like in
+# rust-optimizer-arm:0.12.8) you should uncomment this like to be able to
+# compile the contract
+#
+#cargo-features = ["workspace-inheritance"]
+
+[package]
+authors = ["Sunny Aggarwal <sunnya97@protonmail.ch>"]
```

Comment:
This still accurate?

Reply:
Most of swaprouter was written by Sunny. I just added twap, transfer ownership and some cleanup. Maybe we should have a generic authors string? "osmosis team" or something